README.md

```diff
@@ -50,6 +54,96 @@ graph TD
+- **PTAX flag**: The PTAX-203 form is required to be filled out for an Illinois Real Estate Transfer Declaration for non-residential property over $1 million and/or as required by Illinois Department of Revenue. If there are certain fields filled out on this form, we mark the sale with a ptax flag. 
+- **Non-person sale**: We flag a keyword that suggests the sale involves a legal entity (industrial buyer, bank, real estate firm, construction, etc)
+- **Flip Sale**: The owner of the home owned the property for less than 1 year
+- **Anomaly**: Our isolation forest statistical outlier algorithm flagged the sale
+
+The following is a list of all flag types:
+
+### High Price
+
+- **PTAX outlier (high)**: PTAX flag & [1 high statistical outlier type]
+- **Home flip sale (high)**: Short-term owner < 1 year & [1 high statistical outlier type]
+- **Family sale (high)**: Last name match & [1 high statistical outlier type]
+- **Non-person sale (high)**: Legal / corporate entity + [1 high statistical outlier type]
+- **Anomaly (High)**: Anomaly algorithm (high) & [1 high statistical outlier type]
+- **High price (raw & sqft )**: High price & high price per sq. ft. 
+- **High price swing**: Large swing away from mean  & high price outlier 
+- **High price (raw)**: High price 
+- **High price (per sqft)**: High price per sq. ft. 
+
+### Low Price
+
+- **PTAX outlier (high)**: PTAX flag & [1 low statistical outlier type]
+- **Home flip sale (low)**: Short-term owner < 1 year & [1 low statistical outlier type]
+- **Family sale (low)**: Last name match & [1 low statistical outlier type]
+- **Non-person sale (low)**: Legal / corporate entity + [1 low statistical outlier type]
+- **Anomaly**: - Anomaly algorithm (low) & [1 low statistical outlier type]
+- **Low price (raw & sqft )**: Large swing away from mean & low price outlier 
+- **Low price swing**: Low price & low price per sq. ft. 
+- **Low price (raw)**: Low price (or under $10k)
+- **Low price (per sqft)**: Low price per sq. ft.
+
+## Distribution of Outlier Types
+
+Around **7.2%** of the total sales have some sort of `Outlier` classification.  Within that 7.2%, the makeup of the outlier distribution is approximately as follows:
+
+|Outlier Type     |Proportion|
+|-----------------------|----------|
+|PTAX-203 flag          |0.401     |
+|Non-person sale (low)  |0.176     |
+|Non-person sale (high) |0.081     |
+|Anomaly (high)         |0.063     |
+|High price (raw)       |0.058     |
+|Low price (raw)        |0.054     |
+|Low price (raw & sqft) |0.052     |
+|Low price (sqft)       |0.021     |
+|Anomaly (low)          |0.02      |
+|High price (sqft)      |0.019     |
+|Home flip sale (high)  |0.019     |
+|High price (raw & sqft)|0.016     |
+|Home flip sale (low)   |0.014     |
+|Family sale (low)      |0.006     |
+|Family sale (high)     |0.001     |
+|Low price swing        |~0.0       |
+|High price swing       |~0.0       |
```

Comment:
**issue:** If you're going to include a table like this in the README, I recommend you also include the code required to rebuild it, since this could change significantly if we add new types, alter bounds, etc.

It might be prudent to create a Quarto README that has README.md as an output. You can see an example of this in any of the R packages i.e. https://github.com/ccao-data/assessr/blob/master/README.Rmd.

Note that you can use Python in a quarto doc, no need to build the table using R.

Reply:
Added query in html comment.